app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
-        - gson: <a href='https://github.com/google/gson/blob/master/LICENSE'>Apache-2.0</a>\n\n
+        - Gson: <a href='https://github.com/google/gson/blob/master/LICENSE'>Apache-2.0</a>\n\n
         - MaxMind DB Reader: <a href='https://github.com/maxmind/MaxMind-DB-Reader-java/blob/main/LICENSE'>Apache-2.0</a>\n\n
-        - Android-Flag-Kit: <a href='https://github.com/madebybowtie/FlagKit/blob/master/LICENSE'>MIT</a>\n\n
+        - FlagKit: <a href='https://github.com/madebybowtie/FlagKit/blob/master/LICENSE'>MIT</a>\n\n
         - IP Geolocation by <a href='https://db-ip.com'>DB-IP</a>\n\n
-        - Font Awesome: <a href='https://fontawesome.com/license/free'>License</a>\n\n
-        - App icon made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">flaticon</a>\n\n
+        - Font Awesome: <a href='https://fontawesome.com/license/free'>Licenses</a>\n\n
+        - App icon by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">Flaticon</a>\n\n
         </string>
 
     <!-- Localized -->
     <string name="start_button">Start</string>
     <string name="stop_button">Stop</string>
     <string name="title_activity_settings">Settings</string>
-    <string name="collector_info">UDP Collector: %1$s:%2$d</string>
-    <string name="http_server_status">HTTP Server: http://%1$s:%2$d</string>
+    <string name="collector_info">UDP collector: %1$s:%2$d</string>
+    <string name="http_server_status">HTTP server: http://%1$s:%2$d</string>
     <string name="rcvd_and_sent">%1$s received — %2$s sent</string>
     <string name="query">Query</string>
     <string name="host">Host</string>
     <string name="capture_not_running">Capture not running!</string>
-    <string name="capture_not_started">Capture not started!</string>
+    <string name="capture_not_started">Start capturing network data first.</string>
     <string name="app_filter">Target app</string>
     <string name="ready">Ready</string>
     <string name="about">About</string>
     <string name="status">Status</string>
     <string name="connections_view">Connections</string>
     <string name="no_connections">No connections</string>
-    <string name="connection_details">Connection Details</string>
+    <string name="connection_details">Connection details</string>
     <string name="app">App</string>
     <string name="protocol">Protocol</string>
     <string name="source">Source</string>
     <string name="destination">Destination</string>
     <string name="bytes">Bytes</string>
     <string name="packets">Packets</string>
     <string name="duration">Duration</string>
-    <string name="http_server">HTTP Server</string>
-    <string name="udp_exporter">UDP Exporter</string>
+    <string name="http_server">HTTP server</string>
+    <string name="udp_exporter">UDP exporter collector</string>
```

Comment:
Adding collector here avoids duplicating it in entries, but I don't know if it works. Could it be explained?

Reply:
exporter is the sending side, collector is the receiving side, I would not change them now.